llvm/lib/ToolDrivers/llvm-lib/Options.td

```diff
@@ -23,6 +23,7 @@ def libpath: P<"libpath", "Object file search path">;
 def lst    : F<"list">, HelpText<"List contents of .lib file on stdout">;
 def out    : P<"out", "Path to file to write output">;
 def deffile : P<"def", "def file to use to generate import library">;
+def nativedeffile : P<"defArm64Native", "def file to use to generate native ARM64 symbols in ARM64X import library">;
```

Comment:
`ARM64X` or `ARM64EC`?

Reply:
I wanted to underline the fact that such libraries can be used by both ARM64EC and pure ARM64 target. Thinking about it again, using ARM64X for static libraries may be misleading, it's not exactly an analogue of ARM64X PE images. I will change it, thanks.